core/src/syn/v1/stmt/relate.rs

```diff
@@ -80,6 +82,13 @@ fn relate_i(i: &str) -> IResult<&str, (Value, Value)> {
 	Ok((i, (kind, from)))
 }
 
+fn content(i: &str) -> IResult<&str, Data> {
+	let (i, _) = tag_no_case("CONTENT")(i)?;
+	let (i, _) = shouldbespace(i)?;
+	let (i, v) = alt((into(array), into(param)))(i)?;
+	Ok((i, Data::ContentExpression(v)))
+}
+
```

Comment:
This function already exists and is called inside of data so it is probably not required.

Reply:
Yeah, I was confused about the `into(param)` which does not actually have the same type I think. Will tinker some more, but at least we have the fix for now